brion/blueConfig.h

```diff
@@ -74,6 +74,12 @@ class BlueConfig : public boost::noncopyable
                                       const std::string& sectionName,
                                       const std::string& key ) const;
 
+    /**
+     * @return the user and circuit file target parsers.
+     * @version 1.7
+     */
+    BRION_API Targets getTargetParsers() const;
```

Comment:
I don't agree with the misleading statement. The function returns Targets, which is a vector of brion::Target. Introducing the parser notion is misleading to me. You should keep it if you find somebody else agreeing with you :p